src/net.cpp

```diff
@@ -1171,14 +1169,15 @@ void CConnman::ThreadSocketHandler()
                     // hold in disconnected pool until all refs are released
                     pnode->Release();
                     vNodesDisconnected.push_back(pnode);
+                    continue;
                 }
+                it++;
```

Comment:
Nit: ++it? (make no code difference probably, but I think it's good practice, and avoids later patches changing it around)

Reply:
Ok.